packages/tests-e2e/src/ui/b1-04/all-browsers/SocialProvider-functionality.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { expect, test } from '../../../fixtures/UISignupTest';
+import { microsoftEmail, microsoftPassword, ScreenNames } from '../../../utils/constants';
+
+test.describe('Signup with Microsoft proper user behavior', () => {
```

Comment:
please explain in a comment here, why we only test with microsoft (and not with Google and GitHub)